roles/kubernetes-apps/csi_driver/cinder/tasks/cinder-credential-check.yml

```diff
@@ -24,11 +47,13 @@
     msg: "one of cinder_tenant_id or cinder_tenant_name must be specified"
   when:
     - cinder_tenant_id is not defined or not cinder_tenant_id
-    - cinder_tenant_name is not defined
+    - cinder_tenant_name is not defined or not cinder_tenant_name
+    - cinder_application_credential_name is not defined or not cinder_application_credential_name
 
-- name: Cinder CSI Driver | check cinder_tenant_name value
+- name: Cinder CSI Driver | check cinder_domain_id value
   fail:
-    msg: "one of cinder_tenant_id or cinder_tenant_name must be specified"
+    msg: "one of cinder_domain_id or cinder_domain_name must be specified"
```

Comment:
Why do we need to replace cinder_tenant_id/name with cinder_domain_id/name?
I cannot get the point from the pull request message.

Reply:
So before this PR tenant_id/name was separated into separate steps, but I merged them to one and added this one for the domain_id/name which weren't checked at all.

So it looks like I replaced tenanat to domain, but that's just how git is interpreting it. I have basically just made this one more similar to [this file](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/external_cloud_controller/openstack/tasks/openstack-credential-check.yml)

Reply:
Thanks for your explanation, I got the point.